source/_components/notify.pushover.markdown

```diff
@@ -43,3 +43,24 @@ This is a quote from the Pushover website regarding free/open source apps:
 When setting up the application you can use this [icon](https://home-assistant.io/images/favicon-192x192.png).
 
 To use notifications, please see the [getting started with automation page](/getting-started/automation/).
+
+
+When sending a notification, optional parameters can also be set as per the pushover [API documentation](https://pushover.net/api). 
+
+Example notification triggered from the Alexa component for an intents is shown below which also uses [Automation Templating](https://home-assistant.io/getting-started/automation-templating/) for the message:
+
+```yaml
+# Example configuration.yaml entries
+alexa:
+  intents:
+    LocateIntent:
+      action:
+        service: notify.notify
+        data_template:
+          message: "The location of {{ User }} has been queried via Alexa."
```

Comment:
`{{ User }}` is a template and will be empty after the site is rendered. Jekyll is also using Jinja2.

Change this to 

```
{% raw %}{{ User }}{% endraw %}
```

and we should be fine.